### DISCUSSION TITLE
QDialog: option for blurred background

### DISCUSSION THREAD

**Discussion by ShlomoCode** [quasarframework/quasar#16971] [Ideas / Proposals] [answered]
Right now when dialog is open there is a grayish background around it.
It would be useful to have an option to choose the background to be **blurred** effect (to focus the user on the content), something like what is on [this site](https://www.pwabuilder.com):
![CleanShot 2024-03-06 at 05 54 00@2x](https://github.com/quasarframework/quasar/assets/78599753/0bc42543-056b-4c8e-b343-3a11a34b4767)

You can achieve this using the following CSS:
```css
.q-dialog__backdrop {
  backdrop-filter: blur(10px);
}
```

but it seems to me that it would be useful to add a boolean prop of `blur-backdrop` or something similar to enable it for a particular dialog.

**Comment by rstoenescu** [comment, marked as answer]
Will be available in Quasar v2.14.8.

<img width="597" alt="image" src="https://github.com/quasarframework/quasar/assets/4665641/f102a6bb-a807-431d-9415-5cffb50a2482">

With some examples in docs:

<img width="610" alt="image" src="https://github.com/quasarframework/quasar/assets/4665641/b16cfc58-2774-486d-91c0-10f8b835168e">

**Reply by danliyev** [reply]
Is it possible to do that for the `Dialog.create({...})`?
